alot/commands/common.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2011-2012  Patrick Totzke <patricktotzke@gmail.com>
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+from __future__ import absolute_import
+
+from . import Command
+
+from .globals import PromptCommand
+
+
+class RetagPromptCommand(Command):
+
+    """prompt to retag selected thread's or message's tags"""
+    def apply(self, ui):
```

Comment:
you can ignore this issue, codacity just doesn't seem to like the renamed parameter but this is consistent with the rest of the code. Please remove the now obsolete import suggested in the other codacity comment though.
I'll leave the merge up to @dcbaker, because we may want to postpone until the python3 series is finally merged.